.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- Thanks for contributing to our project! We appreciate your time and effort.
+Please fill out the information below to expedite the review and merge of your pull request.
+-->
+
+#### What this PR does / why we need it:
+<!-- A clear description of the problem you are trying to solve -->
+
+
+#### Changes made
+<!-- Outline the specific changes made in this merge request. -->
+
+
+#### Which issue(s) this PR fixes:
+<!--
+Any reference to relevant issue(s).
+Please use the following format, so that the issue will be automatically closed when this PR is merged (see https://help.github.com/articles/closing-issues-using-keywords/)
+
+`Fixes #<issue number>`
+
+If there is not a correspondent issue yet, you might want to open a new one yourself, describing the problem you observed.
+-->
+
+
+#### Test plan
+<!--
+Please, make sure that this PR meets all the necessary quality gates before submitting for review:
+
+- Existing Unit and E2E tests are passing
+- New features should be covered by new Unit and/or E2E tests.
+
+This will help us to ensure that your changes are working as expected and will not break in the future.
+ 
+In order to save cloud resources, we invite you to submit the PR as a draft and run a single E2E test job, e.g. adding a comment to the PR with the following message in order to run E2E test on OCP 4.15 only:
+
+> /test 4.15-openshift-e2e
```

Comment:
are our test names the same everywhere...? 🙈

Reply:
It's just an example, but it can be even more generic

Reply:
or we align the names, would make life easier for everone 🙂

Reply:
I'm confused, isn't `/test x.y-openshift-e2e` the same for all operators? :thinking:

Reply:
`grep -rin context: | grep e2e` under `openshift/release/ci-operator/jobs/mediks8` seems to give the same name everywhere (if I got the origin of CI test names right)